### DISCUSSION TITLE
Error when using "Resume Training" Tab in RAVEv2_Training Google Colab Notebook by Hexorcismos

### DISCUSSION THREAD

**Discussion by chrizzlemadizzle** [acids-ircam/RAVE#255] [Q&A]
The Training process works just fine with my dataset. However, if I stop the training process and resume it by using the "Resume Training"-Tab I get an error, which I can't solve:

![error ravev2 resume training](https://github.com/acids-ircam/RAVE/assets/65629614/c3d218fd-9ef8-493b-8510-0f966c49207d)

I don't understand why `--db_path=None`. 
I double checked all my entries.
I tried to set `$db_path "/content/dataset"` manually in the code.
I tried renaming the "dataset"-directory which was created during former training stage, so that a new one can be created while resuming training, but the same error keeps popping up. 
Does anyone know how to fix this? Thanks!

**Comment by joystiller** [comment]
You need to add a space between the flag and the parameter, that is, before **--db_path**. I think. Open up the code and make sure there are spaces between all the flags.
